sources/packages/web/src/views/Student.vue

```diff
@@ -180,18 +179,17 @@ export default {
     },
   },
   setup(props: any) {
-    // Readonly student data from state.
-    const store = useStore();
     const router = useRouter();
     const toast = useToast();
-
-    const readonlyProfile = computed(() => store.state.student.profile);
```

Comment:
How the read-only information is being loaded during the user creation? Am I am missing something in the new implementation?

Reply:
FIxed it with the PR update